src/main/java/org/opentripplanner/model/plan/Place.java

```diff
@@ -98,12 +93,9 @@ public String toStringShort() {
     public String toString() {
         return ToStringBuilder.of(Place.class)
                 .addStr("name", name)
-                .addObj("stopId", stopId)
-                .addStr("stopCode", stopCode)
-                .addStr("platformCode", platformCode)
+                .addObj("stop", stop)
```

Comment:
Does stop have a nice toString()?

Reply:
Yes, it is a bit more limited, by just outputting the stop id